stroom-util-shared/src/main/java/stroom/util/shared/StreamLocation.java

```diff
@@ -98,7 +98,7 @@ public int compareTo(final Location o) {
 
     @Override
     public String toString() {
-        return streamNo +
+        return (partIndex + 1) +
```

Comment:
maybe change to "partNo: N lineNo: M" to avoid any confusion given the obj is holding indexes